holoviews/streams.py

```diff
@@ -547,6 +547,56 @@ def hashkey(self):
 
 
 
+
+class ParamStream(Stream):
+    """
+    A Stream that watches the changes in the parameters of the supplied
+    Parameterized objects or the dependencies of a method on a Parameterized
+    object.
+    """
+
+    parameterized = param.ClassSelector(class_=param.Parameterized, constant=True, doc="""
+        Parameterized instance to watch for parameter changes.""")
+
+    parameters = param.List([], constant=True, doc="""
+        Parameters on the parameterized to watch.""")
+
+    def __init__(self, parameterized, parameters=None, **params):
+        self.is_method = util.is_param_method(parameterized)
+        if self.is_method:
+            method = parameterized
+            parameterized = util.get_method_owner(parameterized)
+            parameters = [p.name for p in parameterized.param.params_depended_on(method.__name__)]
+        elif parameters is None:
+            parameters = [p for p in parameterized.params()]
+
+        super(ParamStream, self).__init__(
+            parameterized=parameterized,
+            parameters=parameters, **params
+        )
+        for p in self.parameters:
+            self.parameterized.param.watch(self._listener, p)
```

Comment:
I don't see any use of `unwatch` yet. Surely there are situations where clean up is needed?

Reply:
Instead of defining ``_listener``, wouldn't a lambda be sufficient?